Develappers.BillomatNet/InvoiceService.cs

```diff
@@ -142,28 +142,15 @@ public async Task<InvoiceItem> GetItemByIdAsync(int id, CancellationToken token
         /// A task that represents the asynchronous operation.
         /// The task result contains the id of the newly created invoice.
         /// </returns>
-        public async Task<int> CreateAsync (Invoice invoice,  List<InvoiceItem> invoiceItems, CancellationToken token = default(CancellationToken))
+        public async Task<Invoice> CreateAsync (Invoice invoice,  List<InvoiceItem> invoiceItems, CancellationToken token = default(CancellationToken))
```

Comment:
kannst man invoiceItems nicht auch gleich in invoice als Property integrieren? ähnlich wie das bei dem API-Typ der Fall ist